blog/nse_portfolio_sorts/index.qmd

```diff
@@ -0,0 +1,931 @@
+data_premia |>
+  unnest(premium_estimate) |> 
+  ggplot() +
+  geom_density(aes(x  = premium_estimate),
+               alpha = 0.25, 
+               linewidth = 1.2,
+               color = "#3B9AB2",
+               fill = "#3B9AB2") + 
+  labs(x = "Premium (in %, p.m.)",
+       y = NULL)
+```
+
+```{r}
+#| include: false
+#| eval: false
+q25 <- quantile(data_premia$premium_estimate, 0.25)
+q75 <- quantile(data_premia$premium_estimate, 0.75)
+
+highlight <- tibble(x = density(data_premia$premium_estimate)$x,
+                    y = density(data_premia$premium_estimate)$y) |> 
+  filter(x > q25 & x < q75) |> 
+  mutate(y = y - 0.01)
+
+nse_thumbnail <- data_premia |>
+  unnest(premium_estimate) |> 
+  ggplot() +
+  theme_bw() + 
+  theme(panel.border = element_blank(), 
+        panel.grid.major = element_blank(),
+        panel.grid.minor = element_blank(),
+        axis.line = element_line(colour = "black")) +
+  geom_density(aes(x  = premium_estimate),
+               alpha = 0.25, 
+               linewidth = 1.2,
+               color = "#3B9AB2",
+               fill = "#3B9AB2") + 
+  labs(x = NULL,
+       y = NULL) +
+  geom_ribbon(data = highlight,
+              aes(x = x, ymax = y), ymin = 0, fill="#EBCC2A", alpha=0.5) +
+  annotate("text", x = 1.1, y = 1.7, label= "Non-Standard Errors", 
+           color = "#E1AF00", size = 10)  +
+  annotate("text", x = 1.25, y = 1.35, label= "In Portfolio Sorts", 
+           color = "#E1AF00", size = 10)
+
+ggsave(plot = nse_thumbnail,
+       width = 7, 
+       height = 4,
+       filename = "blog/nse_portfolio_sorts/thumbnail.jpg", 
+       bg = "white")
+
+```
+
+You can immediately see one of the results in WWW: There is a lot of variation depending on your choices. However, despite the variation, the premium is always positive. I would argue that this is a pretty strong sign.
+
+# Conclusion
+
+In essence, any paper proposing a new predictor reports one of the specifications we considered here. Then, they run a few robustness checks, varying some choices. However, space constraints severely limit the number of these checks. But how about our robustness test here? It is actually nearly 70,000 robustness tests if you think about it. However, it is just one graph (maybe two if showing the t-statistics as well), which condenses much information in a digestible way. Personally, I find this very convincing and attractive.
+
+Of course, we have yet to test the return differentials' time series against some factor model to show that existing factors do not explain the premia. I did not show you this step because the implementation is just an addition to our existing code. You can try it yourself.
+
+If you want to learn more about non-standard errors in portfolio sorts, I refer you to WWW. We not only cover many variables there but also dive much deeper into understanding the variation itself. 
+
```

Comment:
Hier schlägt GPT-4 vor :) 

In summary, each research paper introducing a new predictor often presents one of the specifications we've explored here, followed by several robustness checks, limited by space constraints. However, our analysis could be viewed as nearly 70,000 robustness tests, condensed into an easily interpretable graph or two (if we include t-statistics). Personally, I find this approach both compelling and efficient.

Naturally, we still have to scrutinize the time series of return differentials against a factor model to demonstrate that existing factors can't account for the premiums. I didn't include this step in our discussion as it's a straightforward extension to the code we've used. I encourage you to give it a try.

For those interested in delving deeper into non-standard errors in portfolio sorts, I recommend the WWW paper. It provides a comprehensive overview of numerous variables and a more profound understanding of the variation itself.